pkg/setting/setting_unified_alerting.go

```diff
@@ -63,26 +64,50 @@ type UnifiedAlertingSettings struct {
 	EvaluationTimeout              time.Duration
 	ExecuteAlerts                  bool
 	DefaultConfiguration           string
-	Enabled                        bool
+	Enabled                        *bool
```

Comment:
How come this is a `*bool` when both `nil` and `true` mean Unified Alerting is enabled? Should we just set the value of `Enabled` to `true` in `defaults.ini` and `sample.ini`?

Reply:
That will not work. The function IsEnabled is just a convenience function to hide pointer to bool in the services that use it. The nil at that point should not exist because it is defined in the migration flow.
Settings parsing and migration use the value directly and distinguish between nil and pointer to a value.